gcc/d/d-target.def

```diff
@@ -28,24 +28,31 @@ HOOK_VECTOR (TARGETDM_INITIALIZER, gcc_targetdm)
 #undef HOOK_PREFIX
 #define HOOK_PREFIX "TARGET_"
 
-/* Environmental version identifiers relating to the target OS.  */
+/* Environmental version identifiers relating to the target CPU.  */
 DEFHOOK
-(d_os_builtins,
- "Declare predefined version identifiers",
+(d_cpu_versions,
+ "Declare all environmental version identifiers relating to the target CPU\
+ using the function @code{builtin_version}, which takes a string representing\
+ the name of the version.  Version identifiers predefined by this hook apply\
+ to all modules and being compiled and imported.",
  void, (void),
  hook_void_void)
 
-/* Environmental version identifiers relating to the target CPU.  */
+/* Environmental version identifiers relating to the target OS.  */
 DEFHOOK
-(d_cpu_builtins,
- "Declare predefined version identifiers",
+(d_os_versions,
+ "Similarly to @code{TARGET_D_CPU_VERSIONS}, but is used for versions relating\
+ to the target operating system.",
  void, (void),
  hook_void_void)
 
-/* The size of OS CRITICAL_SECTION or pthread_mutex_t.  */
+/* The sizeof CRITICAL_SECTION or pthread_mutex_t.  */
 DEFHOOK
 (d_critsec_size,
- "Declare size of critical section",
+ "Returns the size of the data structure used by the targeted operating system\
+ for critical sections and monitors.  For example, on Microsoft Windows this\
+ would return the @code{sizeof(CRITICAL_SECTION)}, while other platforms that\
+ implement pthreads would return @code{sizeof(pthread_mutex_t)}.",
```

Comment:
AFAICS it should be `D_CRITICAL_SECTION.sizeof` which includes an additional pointer field?